README.adoc

```diff
@@ -85,6 +85,22 @@ That Makefile target takes care of several steps that can be executed separately
 * create `CatalogSource` and `ConfigMap` with `ClusterServiceVersion` and all CRDs in the `openshift-marketplace`: `$ make deploy-csv`
 * and as the last step the actual installation via creating `OperatorGroup` and `Subscription` in the test namespace.
 
+== Releasing operator
+
+The releases of the operator are automatically managed via Tekton pipeline running under `codeready-toolchain` user account in CRT cluster.
+
+The definition of the CD pipeline can be found here https://github.com/codeready-toolchain/toolchain-cicd.
+The pipeline triggers Makefile targets located in this repository.
+
+=== Broken release
+
+If there is any broken release that cannot be built & pushed through the pipeline and has to be done/fixed manually then do the following steps:
```

Comment:
Would it be worth mentioning one example like `Invalid bundle toolchain-member-operator.v0.0.204-commit-a765b6a, bundle specifies a non-existent replacement toolchain-member-operator.v0.0.203-commit-d24cbac` due to skipped image builds due to OLM bundle limitation?

Reply:
done in 6141f81